src/converters/lintConfigs/rules/ruleConverters/only-arrow-functions.ts

```diff
@@ -17,6 +17,11 @@ export const convertOnlyArrowFunctions: RuleConverter = (tslintRule) => {
         rules: [
             {
                 ...(notices.length !== 0 && { notices }),
+                ruleArguments: [
+                    {
+                        allowStandaloneDeclarations: true,
```

Comment:
I poked through the issue (what a throwback!) and https://github.com/TristonJ/eslint-plugin-prefer-arrow/issues/16 as you'd linked - thanks for the info. It look to me like `allowStandaloneDeclarations` should only be output if `"allow-declarations"` is in the original rule. And we can get rid of the _`"ESLint does not support allowing standalone function declarations."`_ notice.